packages/app/app/stacks/page.tsx

```diff
@@ -37,48 +27,7 @@ export default async function Page() {
           Create New Stack
         </ButtonLink>
       </div>
-      <div className="space-y-6">
-        <Tab.Group>
-          <Tab.List>
-            <div className="flex space-x-2">
-              <Tab
-                as="button"
-                className="px-3 py-1.5 font-semibold rounded-lg ui-selected:bg-surface-75 ui-not-selected:text-em-low outline-none active:ring-2 active:ring-gray-100"
-              >
-                Active Stacks
-              </Tab>
-              <Tab
-                as="button"
-                className="px-3 py-1.5 font-semibold rounded-lg ui-selected:bg-surface-75 ui-not-selected:text-em-low outline-none active:ring-2 active:ring-gray-100"
-              >
-                Completed stacks
-              </Tab>
-            </div>
-          </Tab.List>
-          <Tab.Panels>
-            <Tab.Panel>
-              {hasActiveOrders ? (
-                <StacksTable stackOrders={activeOrders} />
-              ) : (
-                <EmptyStacks text=" No active stacks" />
-              )}
-            </Tab.Panel>
-            <Tab.Panel>
-              {hasCompletedOrders ? (
-                <StacksTable stackOrders={completedOrders} />
-              ) : (
-                <EmptyStacks text="No completed stacks" />
-              )}
-            </Tab.Panel>
-          </Tab.Panels>
-        </Tab.Group>
-      </div>
+      <StackTabs stackOrders={stackOrders} />
```

Comment:
I would prefer to have the tabs exposed and not on a component.
I think that logic fits very well on this page. 

Also don't think that `StacksModal` belongs there. We might used it elsewhere, not related to the tabs.

I would revert this changes.

Reply:
The thing is the Stack tabs use Headless UI, and Headless UI is a client-side only package. Using Headless-UI in a server component will trigger errors for hydration issues. And I wouldn't transform Nextjs routing pages (so far the Home page with the stackbox and the Stacks page with the stacks tabs) into client components.

To remove these warnings I abstracted the logic dependant on Headless packages into their own components.

More info: https://github.com/tailwindlabs/headlessui/issues/2021

We can revert this change and keep the warnings or transform this page into a client-side component, but IMHO these two options are sub-optimal :c

Reply:
I'm okay with this change if it improves the hydration issues and DEX.

But let's rename it to `StackOrders`. 

Having tabs or not is not as important on the name, I want to know it's StackOrders and then if I go to the component I can see we filter by active/complete and we use tabs.

are you okay with this change?

Also would me keep `stack-modal` folder and related components there.

Reply:
Yeah, I'm ok with that. 

Quick question, want me to move the newly named `StackOrders` into `@/components/StackOrders` as a standalone component?

Reply:
Yes, I think it can be like that.

Reply:
Oh, already splited that logic in my PR, I have [StackOrders component](https://github.com/SwaprHQ/stackly-ui/pull/33/files#diff-56341ad50f06b78bbb4e3ed75ca24713793fa0c41972c0980e91e2b8e6bcf688R22-R54).

Maybe I can move it to other folder, and you revert the changes here.

Reply:
Also copied your logic on `activeOrders/completedOrders`.

Reply:
Reverting these changes so we can have them from your work :D

Regarding if I keep getting errors:

For this file I get either a ["no async server components"](https://nextjs.org/docs/messages/no-async-client-component) if I keep the `"use client"` directive in this file.

If I move the `"use client"` directive out of this page component and into the `StacksTable` component, I get a crashing error while building the app, because we're trying to use `Tabs` from Headless UI (client-only lib) in the page component.
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/15f3d9f1-65dd-4bdb-8331-96545e2234b1)

I could solve these errors by abstracting the component that implements `Tabs` into its own component having the `"use client"` directive in there.